components/public-api/gitpod/v1/scm.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package gitpod.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
+
+service SCMService {
```

Comment:
Just so we are all aware: For all these methods we currently imply the authorizedUser. So there will be no way to call any of these methods in the context of a third user. I think. that is fine.

Reply:
Also in case we later want to allow some of these methods to be called for other users, we can introduce a userId as an optional argument.